agent/utils.py

```diff
@@ -56,17 +59,30 @@ def prepare_target(message: m.Message) -> tuple[str, int]:
         raise NotImplementedError
 
 
-def initialize_msf_rpc() -> msfrpc.MsfRpcClient:
-    """Start msfrpcd and connect to it
+def check_msfrpcd() -> None:
```

Comment:
A check method is expected to return boolean the method here is starting msfrpcd so I would name it start_msfrpcd